issue-templates/forms/general/feature-request.yml

```diff
@@ -0,0 +1,74 @@
+# Source: https://github.com/arduino/tooling-project-assets/blob/main/issue-templates/forms/platform-dependent/bug-report.md
+# See: https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+
+name: Feature request
+description: Suggest an enhancement to this project.
+labels:
+  - "type: enhancement"
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the request
+    validations:
+      required: true
+  - type: textarea
+    id: current
+    attributes:
+      label: Describe the current behavior
+      # TODO: Replace TODO_PRODUCT_NAME with the product name of the project.
+      description: |
+        What is the current behavior of TODO_PRODUCT_NAME in relation to your request?
+        How can we reproduce that behavior?
+    validations:
+      required: true
+  - type: input
+    id: project-version
+    attributes:
+      # TODO: Replace TODO_PRODUCT_NAME with the product name of the project.
+      label: TODO_PRODUCT_NAME version
+      description: |
+        Which version of TODO_PRODUCT_NAME are you using?
+        *This should be the most recent version available.*
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating system
+      description: Which operating system(s) are you using on your computer?
+      multiple: true
+      options:
+        - Windows
+        - Linux
+        - macOS
+        - N/A
+    validations:
+      required: true
+  - type: input
+    id: os-version
+    attributes:
+      label: Operating system version
+      description: Which version of the operating system are you using on your computer?
+    validations:
+      required: true
```

Comment:
Is this field mandatory?

Reply:
Thanks. I did not intend the field to be there at all in this file.

This version of the form is for use in projects where the OS is not relevant (there is a separate set of "Platform-dependent issue forms" for the projects where OS is relevant).

I have now removed it: https://github.com/arduino/tooling-project-assets/compare/2bbc89f9ea6fa10cbaf996086ba6ae5dec82ede8..4b8459c07aac2fe418d8fbeb0dba5d68740cdf05